protected/humhub/widgets/CoreJsConfig.php

```diff
@@ -259,7 +259,9 @@ public function run()
                         ]
                     ],
                     'reloadUrl' => Url::to(['/content/content/reload']),
-                    'deleteUrl' => Url::to(['/content/content/delete-id'])
+                    'deleteUrl' => Url::to(['/content/content/delete-id']),
+                    'loadModalUrl' => Url::to(['/content/content/get-admin-delete-modal']),
```

Comment:
@s-tyshchenko The variable name here is a little unclear.
how about `deleteAdminModalUrl` to stay in the current schema? 

Do we really need two new vars here? (can the admin not use the same deleteUrl?)